docs/_config.yml

```diff
@@ -0,0 +1,29 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+title: Mobly
+description: > # this means to ignore newlines until the next key
+  This is the official page for Mobly, a framework for running multiple-device tests.
+baseurl: "/mobly" # the subpath of your site, e.g. /blog
+url: "https://google.github.io"
```

Comment:
google.github.io/mobly seems to take us to a readme-only view. Is that intended?